### ISSUE TITLE
Re-trigger gitstream after submitting a review

### ISSUE THREAD

**Issue by matthewparavati** [linear-b/gitstream#74]
**Is your feature request related to a problem? Please describe.**
After submitting a review, I was thinking that gitstream would be re-triggered but that doesn't seem to be the case.

**Describe the solution you'd like**
After I submit a review, then gitstream should be re-triggered

**Describe alternatives you've considered**
I've looked at the other actions but I don't think they'll help me/I'd run in to a similar issue

**Additional context**
My use case for above was specifically for when a review is submitted with requested changes, I wanted an automation to run to add a "requested changes" label

Below is what I am trying to use but after submitting my review, I didn't see gitstream run again

<details>
<summary>automation snippet</summary>

```
requested_changes:
    if:
      - {{ pr.status | includes(term='requested-changes') }}
    run:
      - action: add-label@v1
        args:
          label: '🚨 Changes requested'
          color: '#EC560D'

```

</details>

**Comment by matthewparavati** [comment]
@vim-zz perfect, thanks! Did you want me to leave this open for now or close and just track the progress of issue 18?

**Comment by matthewparavati** [comment]
@vim-zz Actually, after submitting some more PRs today with requested changes, gitstream seems to be running again, but my automation check always evaluates to false.

I tried updating it to this earlier today just to see if my check was too specific but it's still evaluating to false. 

<details><summary>updated automation snippet</summary>

```
requested_changes:
  if:
    - {{ pr.status | includes(list=['requested-changes', 'Changes requested', 'requested', 'changes', 'Changes']) }}
  run:
    - action: add-label@v1
      args:
        label: '🚨 Changes requested'
        color: '#EC560D'
```

</details>

Couple follow up questions:
- Is it possibly just the case that the PR status isn't actually up to date when the check occurs? Or is something wrong with my automation check?
- Is there a way to check the PR status when the automation is running?

**Comment by matthewparavati** [comment]
@vim-zz any updates/thoughts on this? It always seems to be evaluating to false after submitting a PR review with requested changes

**Comment by vim-zz** [comment]
@matthewparavati we added this to the roadmap, I don't have definite timeline yet

**Comment by vim-zz** [comment]
@matthewparavati: Added:

| Values             | Type      | Description                                              |
|--------------------|-----------|-------------------------------------------------|
| `pr.status` | String | The PR status: `open`, `closed` and **`merged`** |
| `pr.requested_changes` | [String] | List of users that requested changes |

**Comment by matthewparavati** [comment]
@vim-zz thanks for the update. should the `requested_changes` array be zero after changes are resolved or once the user approves the PR?

Basically, we aren't seeing the label getting removed once it's been added and wondering if this is expected?

**Comment by vim-zz** [comment]
@matthewparavati the list clears when all issues are resolved. Do you see gitStream triggers a new check after the changes are resolved? If not, this is a bug. If it tirggers bug doesn't change the resolution it's might be anoter problem

**Comment by matthewparavati** [comment]
@vim-zz I was able to re-test and I did not see a new gitstream check after marking a requested change as resolved, I created #153 as a follow up to this
